keps/sig-scheduling/4247-queueinghint/kep.yaml

```diff
@@ -18,12 +18,12 @@ see-also:
 
 stage: beta
 
-latest-milestone: "v1.29"
+latest-milestone: "v1.32"
 
 milestone:
   alpha: "v1.26" # This KEP stems from /keps/sig-node/3063-dynamic-resource-allocation.
-  beta: "v1.28"
-  stable: "v1.32"
+  beta: "v1.32"
```

Comment:
Actually, do we have to change it to .32? 
Our current situation is that it's marked as beta already, but not enabled by default. (we found the bug and changed the feature gate from enabled to disabled by default)
And, we're planning to make the feature gate enabled by default.

Reply:
So, technically it's already beta (although the flag is disabled by default), and then I'm also not sure we need to make it on the release process like other normal graduations. (I guess Yes we should)